README.md

```diff
@@ -1,7 +1,131 @@
 # Terragrunt and Terratest example
 
-## Pre-requisites
-- [terraform] v0.11.12
-- [terragrunt] v0.18.1
-- [terratest] 
-- [ansible] 2.7.8
+## Overview
+
+This is an example on how to use Terragrunt and Terratest on existing terraform modules. The example includes a terraform module for provisioning a node-red app in EC2.
+Both libraries were developed at Gruntwork.
+
+### Pre-requisites
+- [terraform][terraform_tool] v0.11.12
+- [terragrunt][terragrunt_tool] v0.18.1
+- [terratest][terratest_tool]
+- ansible 2.7.8
+- golang go1.12
+
+2. Supply credentials and config as environment variables
+```bash
+$ export AWS_ACCESS_KEY_ID="anaccesskey"
+$ export AWS_SECRET_ACCESS_KEY="asecretkey"
+$ export TF_VAR_key_name="keynametobeused"
+$ export TF_VAR_ssh_key_private="locationofprivatekey"
+$ export ANSIBLE_HOST_KEY_CHECKING=False
+```
+
+### Terragrunt
+
+One of the motivation for Terragrunt is to provide a way to keep Terraform code DRY. The idea is to define the infrastructure code once and expose input variables that can be use to define different environments (e.g. prod, qa, stage, dev).
```

Comment:
@tianomagdaong could you add a list of features that Terragrunt adds to Terraform. What are the other benefits or using Terragrunt over Terraform directly?